### ISSUE TITLE
Cannot resize map card in a dashboard

### ISSUE THREAD

**Issue by nmcalabroso** [metabase/metabase#12249]
**Describe the bug**
Adding a map to a dashboard works. However, once I attempt to resize the card of the map the resizing function doesn't stop when releasing the mouse or even clicking anywhere outside the card. Note that the map is a custom geojson region map.

**Logs**
```
Uncaught TypeError: this.props.onRenderError is not a function
    at t.value (app-main.bundle.js?a10c9974cc03fead96be:5)
    at t.value (app-main.bundle.js?a10c9974cc03fead96be:5)
    at e.notifyAll (vendor.bundle.js?a10c9974cc03fead96be:1)
    at d.close (vendor.bundle.js?a10c9974cc03fead96be:1)
    at d.closeAll (vendor.bundle.js?a10c9974cc03fead96be:1)
    at d.perform (vendor.bundle.js?a10c9974cc03fead96be:1)
    at v.perform (vendor.bundle.js?a10c9974cc03fead96be:1)
    at v.perform (vendor.bundle.js?a10c9974cc03fead96be:1)
    at M (vendor.bundle.js?a10c9974cc03fead96be:1)
    at v.close (vendor.bundle.js?a10c9974cc03fead96be:1)
```

**To Reproduce**
1. Create a dashboard.
2. Create a Map question.
3. Add it to the dashboard.
4. Try to resize the map card in the dashboard.

**Expected behavior**
Card resizing feature in the dashboard should work

**Screenshots**
<img width="1425" alt="Screen Shot 2020-04-03 at 1 49 23 AM" src="https://user-images.githubusercontent.com/5373999/78281971-1fd25b80-754e-11ea-9108-9e9eabe48fb5.png">

**Information about your Metabase Installation:**

You can get this information by going to Admin -> Troubleshooting.

- Your browser and the version: Chrome Version 80.0.3987.149 (Official Build) (64-bit)
- Your operating system: MacOS Catalina
- Your databases: Postgres
- Metabase version: v0.34.3
- Metabase hosting environment: Docker on Ubuntu
- Metabase internal database: Postgres

**Severity**
It's not gonna stop us from visualizing mapping since we still can click on the actual map that will redirect us to the question. It's just a very annoying bug on an otherwise best data viz platform.

**Comment by nmcalabroso** [comment]
UPDATE: Tried changing the values of `sizeX` and `sizeY` in the `report_dashboardcard` table to 6x6 instead of the default 4x4. Now resizing feature is working fine.

**Comment by flamber** [comment]
Hi @nmcalabroso 
I have a feeling that there's likely more errors in the console than just the one you listed.
There's several reports in #5700 of that specific error - some say it might be an un-clean GeoJSON.
Can you post a link to the GeoJSON file?

**Comment by nmcalabroso** [comment]
@flamber thanks for quick response! Here's the custom geojson file: https://gitlab.com/jdultra/geojsonph/-/raw/master/Philippines/Luzon/Metropolitant%20Manila/MetropolitantManila.minimal.json that we are using.

By the way, manually setting the size in `report_dashboardcard` fixes the UI editing issue. Will try to get more console errors for this bug report.

**Comment by flamber** [comment]
Seems like it's a repeat of #6888, which was fixed by #10931, but I guess we forgot the mouse blocker on custom maps. Giving this P2, since it's really difficult to resize/move the card.

**Comment by paulrosenzweig** [comment]
Fixed in #12307
